db/dataset_eval.py

```diff
@@ -298,6 +298,27 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
     job_id = result.fetchone()[0]
     return job_id
 
+def get_pending_jobs_for_user(user_id):
+    with db.engine.connect() as connection:
+        query = sqlalchemy.text("""
+                     SELECT dataset_eval_jobs.id::text
+                       FROM dataset_eval_jobs
+                       JOIN dataset_snapshot
+                         ON dataset_snapshot.id = dataset_eval_jobs.snapshot_id
+                       JOIN dataset
+                         ON dataset.id = dataset_snapshot.dataset_id
+                      WHERE status = :status
+                        AND eval_location = :eval_location
+                        AND author = :author_id
```

Comment:
I'd prefer to name this placeholder `:user_id`, so that it matches with the parameter name